src/master-playlist-controller.js

```diff
@@ -73,6 +74,19 @@ const parseCodecs = function(codecs) {
 };
 
 /**
+ * Replace codecs in the codec string with the old apple-style `avc1.<dd>.<dd>` to the
+ * standard `avc1.<hhhhhh>`.
+ *
+ * @param codecString {String} the codec string
+ * @return {String} the codec string with old apple-style codecs replaced
+ */
+export const mapLegacyAvcCodecs = function(codecString) {
```

Comment:
Can we add an underscore and @private to this since we don't want this to be a part of the api that we are exposing